src/Illuminate/Foundation/Vite.php

```diff
@@ -397,4 +418,63 @@ public function reactRefresh()
             )
         );
     }
+
+    /**
+     * Merge the given entry points.
+     *
+     * @param  array  $entryPoints
+     * @return $this
+     */
+    public function withEntryPoints(array $entryPoints)
+    {
+        $this->entryPoints = array_merge($this->entryPoints, $entryPoints);
+
+        return $this;
+    }
+
+    /**
+     * Set the hot file path attribute.
+     *
+     * @param  string  $path
+     * @return $this
+     */
+    public function useHotFile($path)
+    {
+        $this->hotFilePath = $path;
+
+        return $this;
+    }
+
+    /**
+     * Get the hot file path.
+     *
+     * @return string
+     */
+    protected function hotFilePath()
+    {
+        return $this->hotFilePath ?? public_path('/hot');
+    }
+
+    /**
+     * Set the build directory attribute.
+     *
+     * @param  string  $path
+     * @return $this
+     */
+    public function useBuildDirectory($path)
+    {
+        $this->buildDirectory = $path;
+
+        return $this;
+    }
+
+    /**
+     * Get content as a string of HTML.
+     *
+     * @return string
+     */
+    public function toHtml()
+    {
+        return $this->__invoke($this->entryPoints, $this->buildDirectory)->toHtml();
+    }
```

Comment:
Based on https://github.com/laravel/framework/pull/43620#issuecomment-1214485027 we can then make this:

```suggestion
    public function toHtml()
    {
        return $this->__invoke($this->entryPoints)->toHtml();
    }
```